internal/containerupdate/exec_updater.go

```diff
@@ -64,3 +69,15 @@ func (cu *ExecUpdater) UpdateContainer(ctx context.Context, cInfo store.Containe
 
 	return nil
 }
+
+func handleK8sExecError(out *bytes.Buffer, err error) error {
+	msg := strings.ToLower(fmt.Sprintf("%s\n%s", out.String(), err.Error()))
+	if strings.Contains(msg, "permission denied") || strings.Contains(msg, "cannot open") {
+		return fmt.Errorf("denied access to the container filesystem. Please check:\n"+
+			"  1) That the container image has a writable file system\n"+
+			"  2) That the container image default user has write access to the file system\n"+
```

Comment:
The output in the screenshot appears to be double-spaced, but that doesn't look to be the intent here. Not a blocker, just a bit eyebrow-raising.

Reply:
ya, i believe that's a feature of the fallback ui, i'm not entirely sure what the thinking behind this is /cc @hyu

Reply:
This design is from a while ago, but I _think_ the intent was to make it very obvious that fallback was happening, and perhaps there were also some bad assumptions about what fallbacks would look like (e.g., not multi-line).

Re-evaluating this now, I think we should just make the fallback message have the same spacing as everything else.

Reply:
adjusting the spacing a bit more, here's what it looks like now:

![Screenshot from 2021-02-18 09-57-05](https://user-images.githubusercontent.com/278641/108375158-b6902100-71cf-11eb-80e3-e190240303eb.png)